packages/eslint-plugin/src/utils/ast-string.tsx

```diff
@@ -0,0 +1,63 @@
+import type {
```

Comment:
Can you rename this from `ast-string` to `ast-to-string`? It does highlight that `buildImportDeclaration` doesn't quite belong in this file either given its input parameter